ironfish/src/blockchain/blockchain.ts

```diff
@@ -1182,8 +1194,9 @@ export class Blockchain {
       yield {
         transaction,
         initialNoteIndex: noteIndex,
-        blockHash: header.hash.toString('hex'),
+        blockHash: header.hash,
         sequence: header.sequence,
+        previousBlockHash: header.previousBlockHash,
```

Comment:
I'm guessing this is related to an upcoming change? I think it's fine if it merges, but doesn't seem directly used or related to this PR currently